### PR TITLE
Demonstrator documentation

### PR DESCRIPTION
**Pull request by rmlearney-digicatapult**
Instructions for running the demonstrator

### CODE REVIEW

File: demo/hyproof-demo.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+This document walks you through the steps required to deliver a complete demonstration of certificate issuance and revocation using the HyProof system
+
+## Basic Demo Prerequisites
+
+You will require `git` and `docker` to be installed on your machine in order to run these locally, and have some familiarity with a command line interface.
+
+In order to be able to reproduce the steps described in this document, you need to have both the back-end API (`sqnc-hyproof-api`) and front-end user interface (`sqnc-hyproof-client`) components running.
+
+The first step is to fetch both of these codebases from Github.
```
Comment (dc-andysign) [inline demo/hyproof-demo.md:11]: My spellchecker is suggesting code base or code-base .. i think the spellchecker is wrong isn't it?

File: demo/hyproof-demo.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+1. `Emma` previously received a data packet from `Heidi` containing:
+
+    - a `cryptographic commitment` to the content of this data
+    - the `start time` and `end time` of `Heidi's` hydrogen production
+    - the total `kWh` of electricity used by `Heidi`
+
+2. `Emma` first checks this additional private data aligns with what has been publicly recorded by `Heidi` on the ledger using the `cryptographic commitment`. This keeps `Heidi` honest
+
+3. `Emma` then calculates the embodied CO~2~ for `Heidi`'s batch from:
```
Comment (dc-andysign) [inline demo/hyproof-demo.md:119]: Is `CO~2~` a special scientific thing ? for some reason this renders as ~~2~~ on my screen ..

File: demo/hyproof-demo.md
Comment (dc-andysign) [inline demo/hyproof-demo.md:1]: Maybe put this in a folder called `./docs/` ?

**Review by dc-andysign — CHANGES_REQUESTED** [review]
This looks awesome .. just added a few things that have a question mark ❓

File: demo/hyproof-demo.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+    - the `time window` sent by `Heidi`
+    - the total `kWh` used by `Heidi`
+    - `Emma`'s knowledge of her `carbon intensity` over that period
+
+4. `Emma` submits this embodied CO~2~ value to the ledger to be added to the certificate that `Heidi` previously intitiated
```
[the remark this answers is not in the document]
Comment (rmlearney-digicatapult) [reply demo/hyproof-demo.md:121]: Am trying to use the tildes to subscript but I think it's dependent on the particular markdown renderer. Might just remove.

File: demo/hyproof-demo.md (hunk 1 above)
Comment (rmlearney-digicatapult) [reply demo/hyproof-demo.md:11]: Seems all are acceptable: https://english.stackexchange.com/questions/237835/codebase-or-code-base

File: demo/hyproof-demo.md (hunk 2 above)
Comment (rmlearney-digicatapult) [reply demo/hyproof-demo.md:115]: Trying to use the tildes ~ to render subscript (read online that's how to do it in md) but seems dependent on the particular md renderer. Might just leave it as CO2

File: demo/hyproof-demo.md
Comment (rmlearney-digicatapult) [reply demo/hyproof-demo.md:1]: Not sure, something to discuss

**Review by dc-andysign — APPROVED** [review]
👍

**Comment by rmlearney-digicatapult** [comment]
Going to merge this documentation without the revocation workflow, and submit a separate PR later for that piece

**Review by dc-andysign — APPROVED** [review]
👍

File: demo/hyproof-demo.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
+# HyProof Demonstration
+
+This document walks you through the steps required to deliver a complete demonstration of certificate issuance and revocation using the HyProof system
+
+## Basic Demo Prerequisites
+
+You will require `git` and `docker` to be installed on your machine in order to run these locally, and have some familiarity with a command line interface.
```
Comment (mattdean-digicatapult) [inline demo/hyproof-demo.md:7]: also need `nodejs`

**Review by mattdean-digicatapult — COMMENTED** [review]
One minor comment on required tools

**Review by dc-andysign — APPROVED** [review]
👍

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**